salt/orchestrate/aws/logging.sls

```diff
@@ -0,0 +1,74 @@
+#!jinja|yaml
+
+# Obtain the grains for one of the elasticsearch nodes
+{% set hosts = [] %}
+{% for host, grains in salt.saltutil.runner(
+    'mine.get',
+    tgt='G@roles:elasticsearch and G@environment:operations', fun='grains.item', tgt_type='compound'
+    ).items() %}
+{% do hosts.append(grains['external_ip']) %}
+{% endfor %}
+
+create_elasticsearch_index_template:
+  elasticsearch_index_template.present:
+    - name: 'logstash-*'
+    - definition:
+        template: 'logstash-*'
+        settings:
+          index.refresh_interval : '5s'
+        mappings:
+          _default_:
+            _all_:
+              enabled: false
```

Comment:
Do we have a particular reason for disabling the __all__ mapping, or is that just a carryover from the old map?